<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>431</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>521</width>
    <height>431</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>521</width>
    <height>431</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QUndoView" name="undoView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="key_img">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>key</string>
    </property>
   </widget>
   <widget class="QLabel" name="user_img">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>user</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>220</width>
      <height>401</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>220</width>
       <height>35</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>input your password </string>
     </property>
    </widget>
    <widget class="QPushButton" name="login">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>220</width>
       <height>35</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signup">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>360</y>
       <width>220</width>
       <height>35</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_img">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>111</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="pwd">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLabel" name="usernames">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Usernames:</string>
     </property>
    </widget>
    <widget class="QLabel" name="havenot">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>340</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Don't have an account?</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>80</width>
       <height>19</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>记住密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>220</width>
       <height>35</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>input your username</string>
     </property>
    </widget>
    <widget class="QLabel" name="judge">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>275</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>judge</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_img2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>260</width>
      <height>380</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="gantanhao">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>162</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>gantanhao</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
